sucker_punch.gemspec

```diff
@@ -22,5 +22,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rake"
   gem.add_development_dependency "pry"
 
-  gem.add_dependency "celluloid", "0.16.0"
+  gem.add_dependency "celluloid", "~> 0.17.1", ">= 0.17.1.2"
```

Comment:
Yup, thank you for handling that! Sorry I couldn't take care of it earlier. Ended up having a very busy last couple of days.